controllers/post.js

```diff
@@ -0,0 +1,6 @@
+exports.getPosts=(req,res)=>{
+    res.json({
+        posts:[{title: "first post"}, {title:"second post"}]
+    });
+    
+}
```

Comment:
@JainishaParmar Please add at the end of line in every file.

CC: @jaimin45 Ashish